dotcom-rendering/src/components/YoutubeAtom/YoutubeAtom.stories.tsx

```diff
@@ -1,974 +1,434 @@
-				]}
-				consentState={consentStateCanTarget}
-				duration={252}
-				format={{
-					theme: Pillar.Culture,
-					design: ArticleDesign.Standard,
-					display: ArticleDisplay.Standard,
-				}}
-				height={450}
-				width={800}
-				shouldStick={true}
-				isMainMedia={true}
-				title="Rayshard Brooks: US justice system treats us like 'animals'"
-				imaEnabled={false}
-				abTestParticipations={{}}
-				adTargeting={adTargeting}
-				imagePositionOnMobile="none"
-				imageSize="large"
-			/>
-			<YoutubeAtom
-				index={789}
-				videoId="3jpXAMwRSu4"
-				alt=""
-				eventEmitters={[
-					(e) => console.log(`analytics event ${e} called`),
-				]}
-				consentState={consentStateCanTarget}
-				duration={252}
-				format={{
-					theme: Pillar.Culture,
-					design: ArticleDesign.Standard,
-					display: ArticleDisplay.Standard,
-				}}
-				height={450}
-				width={800}
-				shouldStick={true}
-				isMainMedia={true}
-				title="Rayshard Brooks: US justice system treats us like 'animals'"
-				imaEnabled={false}
-				abTestParticipations={{}}
-				adTargeting={adTargeting}
-				imagePositionOnMobile="none"
-				imageSize="large"
-			/>
+export const MultipleStickyVideos = {
+	args: {
+		...NoConsent.args,
+		consentState: NoOverlay.args.consentState,
+		shouldStick: true,
+		isMainMedia: true,
+		title: NoOverlay.args.title,
+	},
+	render: (args) => (
+		<div
+			css={css`
+				width: 500px;
+				height: 5000px;
+			`}
+		>
+			<YoutubeAtom {...args} />
+			<YoutubeAtom {...args} index={456} videoId="pcMiS6PW8aQ" />
+			<YoutubeAtom {...args} index={789} videoId="3jpXAMwRSu4" />
 		</div>
-	);
-};
-
-MultipleStickyVideos.parameters = {
-	chromatic: { disableSnapshot: true },
-};
-
-export const PausesOffscreen = (): JSX.Element => {
-	return (
+	),
+	parameters: {
+		chromatic: { disableSnapshot: true },
+	},
+} satisfies Story;
+
+export const PausesOffscreen = {
+	args: {
+		...NoConsent.args,
+		consentState: NoOverlay.args.consentState,
+		isMainMedia: true,
+		title: NoOverlay.args.title,
+		shouldPauseOutOfView: true,
+	},
+	render: (args) => (
 		<div>
 			<div>Scroll down...</div>
-			<YoutubeAtom
-				index={123}
-				videoId="-ZCvZmYlQD8"
-				alt=""
-				eventEmitters={[
-					(e) => console.log(`analytics event ${e} called`),
-				]}
-				consentState={consentStateCanTarget}
-				duration={252}
-				format={{
-					theme: Pillar.Culture,
-					design: ArticleDesign.Standard,
-					display: ArticleDisplay.Standard,
-				}}
-				height={450}
-				width={800}
-				shouldStick={false}
-				isMainMedia={true}
-				title="Rayshard Brooks: US justice system treats us like 'animals'"
-				imaEnabled={false}
-				abTestParticipations={{}}
-				adTargeting={adTargeting}
-				shouldPauseOutOfView={true}
-				imagePositionOnMobile="none"
-				imageSize="large"
-			/>
-			<div style={{ height: '1000px' }}></div>
+			<YoutubeAtom {...args} />
+			<div
+				css={css`
+					height: 1000px;
+				`}
```

Comment:
Why not keep the explicit styles here? For a single declaration it’s more performant and more typesafe.

```suggestion
			<div
				style={{
					height: '1000px;'
				}}
```

Reply:
Regarding object styles, we don't generally use them across the codebase, preferring CSS strings. If we think they're better, e.g. because they offer better type safety, I'd prefer to adopt them universally rather than mixing two different approaches.

Regarding the `style` prop, I suppose these fit one of the definitions of "dynamic styles" in the sense that they're unique to a single element[^1]? But they aren't going to change frequently, as in the other definition. I can change it to `style` if you'd prefer though?

[^1]: https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles

Reply:
Yes indeed, following Emotion best practices means we should use the `style` prop for dynamic style. In this case, whether it’s static or dynamic isn’t very clear cut, but I think understanding which one is favourable is an interesting exercise – although not a bocker at all for this individual PR!